diagnostics-program/startDiag.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 #Run diag
-rm /opt/nebraDiagnostics/html/index.html
-rm /opt/nebraDiagnostics/html/initFile.txt
-#rm /var/data/public_keys
+rm -f /opt/nebraDiagnostics/html/index.html
+rm -f /opt/nebraDiagnostics/html/initFile.txt
+#rm -f /var/data/public_keys
 #cp /opt/nebraDiagnostics/html/index.html.template /opt/nebraDiagnostics/html/index.html
 
 nginx
```

Comment:
@ilyastrodubtsev I'm pretty sure like 12 will never execute as `nginx` won't fork out. Could you confirm?